bigbluebutton-html5/imports/ui/components/video-dock/component.jsx

```diff
@@ -40,19 +41,39 @@ class VideoElement extends Component {
   }
 
   render() {
+    let cssClass;
+    if (this.props.shared || !this.props.localCamera) {
+      cssClass = styles.sharedWebcamVideoLocal;
+    }
+    else {
+      cssClass = styles.sharedWebcamVideo;
+    }
     return (
-      <video
-        id={`video-elem-${this.props.videoId}`}
-        width={320}
-        height={240}
-        autoPlay="autoPlay"
-        playsInline="playsInline"
-      />
+      <div className={styles.videoContainer + " " + cssClass} >
```

Comment:
Please make use of the [`classnames`](https://github.com/JedWatson/classnames) package.